src/Geno/C++/GUI/MainWindow.cpp

```diff
@@ -451,6 +466,16 @@ void MainWindow::ImGuiSettingsWriteAllCB( ImGuiContext* pContext, ImGuiSettingsH
 		}
 	}
 
+	for( int I = static_cast< int >( MainWindow::Instance().GetRecentWorkspaces().size() ) - 1; I >= 0; I-- )
+	{
+		pOutBuffer->appendf( "[%s][%s]\n", pHandler->TypeName, "Recent Workspaces" );
+		pOutBuffer->appendf( "Path=%s\n", MainWindow::Instance().GetRecentWorkspaces()[ I ].string().c_str() );
+		pOutBuffer->append( "\n" );
+
+		if( I == 0 )
+			break;
```

Comment:
Isn't this redundant? The condition `I >= 0` already stops the loop when I goes below 0